api-catalog-services/src/main/java/com/ca/mfaas/apicatalog/gateway/GatewayLookupException.java

```diff
@@ -10,16 +10,16 @@
 package com.ca.mfaas.apicatalog.gateway;
 
 /**
- * Exception used to indicate that the Gateway scheme and host could not been initialized.
+ * Exception thrown when there's need to interrupt {@link GatewayLookupService} retry process
```

Comment:
"...a need...", "interrupt **or** retry a process?"

Reply:
"interrupt or retry a process?" - There's retry process and we are trying to say that you may interrupt it by this exception.